src/Raven.Client/Documents/Subscriptions/SubscriptionCriteria.cs

```diff
@@ -5,13 +5,30 @@
 // -----------------------------------------------------------------------
 
 using System;
+using Raven.Client.Documents.Replication;
+using Raven.Client.Documents.Replication.Messages;
 using Sparrow.Json;
 
 namespace Raven.Client.Documents.Subscriptions
 {
+    public class SubscriptionRaftState:IDatabaseTask
+    {
+        public SubscriptionRaftState()
+        {
+
+        }
+        public SubscriptionCriteria Criteria { get; set; }
+        public ChangeVectorEntry[] ChangeVector { get; set; }
+        public long Etag { get; set; }
```

Comment:
What is the meaning of `Etag` in this context? Is this the `SubscriptionId`, in that case, we are better calling it that.